### ISSUE TITLE
Daily a11y check workflow should inform Slack of outcome

### ISSUE THREAD

**Issue by timcosgrove** [department-of-veterans-affairs/va.gov-cms#17776]
## Requirements

When an a11y check runs, it should inform Slack of success or failure outcome, so that interested parties can be alerted to fresh a11y checks and to problems with it.

```[tasklist]
### Acceptance criteria
- [ ] Upon success or failure, the a11y check messages Slack with that information
```

## Background & implementation details

Slack channel is TBD.
